src/hooks/useInterval.js

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect, useRef } from 'react'
+
+function useInterval(callback, delay) {
+  const savedCallback = useRef()
+  let firstTime = true
+
+  // Remember the latest callback.
+  useEffect(() => {
+    savedCallback.current = callback
+  }, [callback])
+
+  // Set up the interval.
+  useEffect(() => {
+    function tick() {
+      if (!firstTime)
```

Comment:
Either some weird braces or bad code here. First time should be set to `false` on the first run of `tick()` and it should not run the callback on the first run of `tick()`.